src/components/comment-input.component.js

```diff
@@ -113,12 +120,12 @@ export class CommentInput extends Component {
                   : translate('issue.main.commentInput', language)
               }
               multiline
-              blurOnSubmit
+              blurOnSubmit={false}
               onChangeText={text => this.setState({ text })}
               onContentSizeChange={event =>
                 this.setState({ height: event.nativeEvent.contentSize.height })}
               onSubmitEditing={event =>
```

Comment:
It seems I figured out what I did wrong: TexInput does not have the `onSubmit` handler, and I just changed `onSubmitEditing` to `onSubmit`, but it's strange, then it should not work at all, in the sense that the sending should not happen at all, but Android works.

I'll check it again later.